automation/alpha-perf-test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Simple script to start perf test in alpha
+echo "Starting Perf test in alpha"
+echo "FOR EASE OF USE: IF YOU RUN THIS SCRIPT AND A BROWSER OPENS WITH GOOGLE LOGIN, KILL THE SCRIPT AND GCLOUD AUTH INTO EACH USER FIRST (see script comments)"
+
+# Log in with each user:
+# gcloud auth login dominique.testerson@gmail.com
+# gcloud auth login gary.testerson1@gmail.com
+# gcloud auth login felicity.testerson@gmail.com
+# gcloud auth login frida.testerson@gmail.com
+# gcloud auth login elvin.testerson@gmail.com
+# gcloud auth login test.firec@gmail.com
+
+
+echo "Logging in with dominique.testerson@gmail.com"
+gcloud auth login dominique.testerson@gmail.com
```

Comment:
This whole thing, from here to the curl, should probably be a function that you just call with the email address to use.

Reply:
good point. every curl is against a different workspace, but this could be made into a function